python/tvm/micro/testing/evaluation.py

```diff
@@ -94,6 +95,15 @@ def create_aot_session(
     executor = tvm.relay.backend.Executor("aot")
     crt_runtime = tvm.relay.backend.Runtime("crt", {"system-lib": True})
 
+    if use_cmsis_nn:
+        with tvm.transform.PassContext(
+            config={
+                "tir.disable_vectorize": True,
+                "relay.ext.cmsisnn.options": {"mcpu": target.mcpu},
+            }
+        ):
+            mod = cmsisnn.partition_for_cmsisnn(mod, params, mcpu=target.mcpu)
```

Comment:
Let's put this into the `ExitStack()` block below - remove likes 98-106 and insert:
```
mod = cmsisnn.partition_for_cmsisnn(mod, params, mcpu=target.mcpu)
```
in the `if use_cmsiss_nn` block